Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.6.2-alpine
+RUN apk add build-base tzdata sqlite sqlite-dev && gem install tzinfo-data
+WORKDIR /app
+COPY Gemfile /app/Gemfile
+COPY Gemfile.lock /app/Gemfile.lock
+RUN bundle install --deployment --without development test
+COPY . /app
```

Comment:
Do we really want the whole git repository within our app? It would make the container unnecessary big